src/frontend/src/common/components/library/data_subview/components/CreateNSTreeModal/index.tsx

```diff
@@ -119,6 +133,37 @@ export const CreateNSTreeModal = ({
     USER_FEATURE_FLAGS.tree_location_filter
   );
 
+  const handleFilterChange = (setFilter: () => void): void => {
```

Comment:
Nit: This is minor, but I think I'd prefer the name of the arg to be `onChange` or something else a bit more generic? It took me awhile to realize `setFilter` wasn't actually setting some filter it was being given, it was just a callback function and the intention was it to be a callback function that would handle the setting of filters.